docs/sources/developers/plugins/create-a-grafana-plugin/develop-a-plugin/build-a-data-source-backend-plugin.md

```diff
@@ -96,6 +96,14 @@ You now have a new data source instance of your plugin that is ready to use in a
 
 #### Grafana doesn't load my plugin
 
+Ensure that Grafana has been started in development mode. If you are running Grafana from source, you'll need to include the add the following line to your `conf/custom.ini` file (if you don't have one already, go ahead and create this file before proceeding):
+
+```ini
+app_mode = development
+```
+
+You can then start Grafana in development mode by running `mage -v && yarn start` in the repository root. If you are running Grafana from a binary or inside a Docker container, you can start it in development mode by setting the environment variable `GF_DEFAULT_APP_MODE` to `development`.
```

Comment:
I am a bit confused by this. Are you telling the plugin developer to run `mage -v && yarn start` in Grafana's source code directory, or in the plugin's directory?

If you're referring to Grafana, then the correct command to compile and run Grafana from source should be:

```
make run & make run-frontend
```

(we do not use `Mage` in Grafana core).

If you're referring to the plugin's folder, then the correct command should be:

```
mage -v & yarn dev
```

(this watches and re-builds both the plugin's backend and frontend at the same time, notice `&` instead of `&&`)

I would also separate those into two separate paragraphs (but would also like to hear feedback from someone in the docs squad regarding this one in particular):

```suggestion
You can then start Grafana in development mode by running `mage -v && yarn start` in the repository root.

If you are running Grafana from a binary or inside a Docker container, you can start it in development mode by setting the environment variable `GF_DEFAULT_APP_MODE` to `development`.
```

Reply:
@xnyo Thanks for catching this, this was corrected in 875ba75. I've also gone ahead and broken the paragraph into two, as I think that you're right, it scans better this way.